### PR TITLE
Adds verbose statements & other improvements

### PR DESCRIPTION
**Pull request by ibraheem-opentensor**
- Adds handler for console.status in `print_verbose`
- Forces use_password as default when creating hotkeys
- Updates senate table
- Removes few extra calls

### CODE REVIEW

File: bittensor_cli/src/commands/wallets.py [hunk 1]
```diff
@@ -514,15 +519,19 @@ async def overview(
...
+        print_verbose("Fetching total balance for hotkeys", status)
         block_hash = await subtensor.substrate.get_chain_head()
         all_hotkeys, total_balance = await _get_total_balance(
             total_balance, subtensor, wallet, all_wallets
         )
 
         # We are printing for a select number of hotkeys from all_hotkeys.
         if include_hotkeys:
+            print_verbose(
+                "Fetching for select hotkeys passed in 'include_hotkeys'", status
```
Comment (thewhaleking) [inline bittensor_cli/src/commands/wallets.py:533]: Does this work?

File: bittensor_cli/src/commands/wallets.py [hunk 2]
```diff
@@ -532,11 +541,13 @@ async def overview(
 
         # Pull neuron info for all keys.
         neurons: dict[str, list[NeuronInfoLite]] = {}
+        print_verbose("Fetching subnet netuids", status)
```
Comment (thewhaleking) [inline bittensor_cli/src/commands/wallets.py:544]: Same here

**Review by thewhaleking — APPROVED** [review]

File: bittensor_cli/src/bittensor/utils.py [hunk 3]
```diff
@@ -26,7 +26,19 @@
...
 err_console = Console(stderr=True)
 verbose_console = Console(quiet=True)
-critical_console = Console(quiet=False)
+
+
+def print_verbose(message: str, status=None):
+    """Print verbose messages while temporarily pausing the status spinner."""
+    if status:
+        # Pause the spinner to avoid overlapping
+        status.stop()
```
Comment (thewhaleking) [inline bittensor_cli/src/bittensor/utils.py:35]: This is smart.

File: bittensor_cli/src/commands/wallets.py (hunk 1 above)
Comment (ibraheem-opentensor) [reply bittensor_cli/src/commands/wallets.py:533]: Yes! 
image

File: bittensor_cli/src/commands/wallets.py (hunk 2 above)
Comment (ibraheem-opentensor) [reply bittensor_cli/src/commands/wallets.py:544]: And yes, working good
image

**Pull request merged**